nodejs/docs/sequelize/specific_details.md

```diff
@@ -0,0 +1,72 @@
+# Sequelize specific details
+
+## Validations in bulk methods
+
+Suppose you have a User model in Sequelize and you have a validation inside.
```

Comment:
"Let's suppose you have an User model in Sequelize with a validation inside it."